deployment/terraform/modules/docker_host/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "docker_hostname" {
+  value = "${aws_instance.docker_host.public_ip}"
```

Comment:
Down the road, yes: It goes into the envvar that's another part of this PR. Not automatic right now, so not strictly necessary, but it will need to be added eventually, so leave it for now, unless you object?

Reply:
Right, the output is useful. I was referring to the `public_ip` part. Resources inside a VPC should not be Internet accessible in general unless it is something like a web server.
https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Scenario2.html#Configuration-2